readme.md

```diff
@@ -60,8 +60,16 @@ camelCase(['__foo__', '--bar'], {pascalCase: true});
 camelCase(['foo', 'BAR'], {pascalCase: true, preserveConsecutiveUppercase: true})
 //=> 'FooBAR'
 
+camelCase('lorem-ipsum') // on a platform with 'en-US'
 camelCase('lorem-ipsum', {locale: 'en-US'});
 //=> 'loremIpsum'
+
+camelCase('lorem-ipsum') // on a platform with 'tr-TR'
+camelCase('lorem-ipsum', {locale: 'tr-TR'});
+//=> 'loremİpsum'
+
+camelCase('lorem-ipsum', {locale: false}); // ignore platform locale
+//=> 'loremIpsum'
```

Comment:
I don't think these additions are needed. It's enough to document it at the API docs for the option.